src/main/resources/db/changelog/data/financial_transaction_categories.csv

```diff
@@ -0,0 +1,10 @@
+id,name,creation_date,transaction_type
+1,shopping,2023-02-23T23:47:52.595721658,EXPENSES
```

Comment:
According to `FinancialTransactionType` **[1]**, the financial transaction type is [ INCOME | EXPENSE ]. Similarly, the types of Financial Transaction Categories are the same.
These are singular nouns.

**[1]** https://github.com/bycza-zagroda/track-expenses-app-backend/blob/853bed3c414cb5e59332f21d944a507b727d3e9a/src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionType.java#L4

Reply:
Ok, so type of financial transactions in financial_transactions.csv is incorrect. I will correct this file right away